src/pfe/Dockerfile_x86_64

```diff
@@ -51,9 +51,9 @@ RUN curl -f -o buildah.rpm $BUILDAH_RPM \
     && cp -r /tmp/$OPENJ9_JRE /opt/java/jre \
     && rm -rf /tmp/$OPENJ9_JRE /tmp/OpenJDK8U-jre_x64_linux_openj9_8u222b10_openj9-0.15.1.tar.gz \
 # Install the latest version of yq
-    && LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/mikefarah/yq/releases/latest) \
-    && LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/') \
-    && curl -sL -O https://github.com/mikefarah/yq/releases/download/$LATEST_VERSION/yq_linux_amd64 \
+    && VERSION=2.4.1 \
```

Comment:
maybe rename this to `YQ_VERSION`, other than that LGTM

Reply:
👍